### ISSUE TITLE
SegFault using Onnxsim embedded in a script

### ISSUE THREAD

**Issue by Wheest** [daquexian/onnx-simplifier#39]
I am looking at embedded Onnx sim in a script, rather than using the command line approach described in the README.

However, in this small script I have made, following the same code as [onnxsim/__main__.py](https://github.com/daquexian/onnx-simplifier/blob/master/onnxsim/__main__.py), but fails with a Segmentation fault.

It is successful when running `python3 -m onnxsim inception_v3.onnx inception_v3_opt.onnx`.

You can see [the script here](https://gist.github.com/Wheest/30856150c46e95b5739342d0360461cb).

Any idea why there would be a segfault when using Onnxsim in this way?

**Comment by daquexian** [comment]
Thanks for your information! I'll check it soon

**Comment by daquexian** [comment]
Sorry that I cannot reproduce the error you met. I didn't get segfault but another error "onnxruntime.capi.onnxruntime_pybind11_state.Fail: [ONNXRuntimeError] : 1 : FAIL : This is an invalid model. Error: Duplicate definition of name (829)" instead, which looks like a PyTorch bug.

However, I guess the segfault you met may be solved by upgrading your pytorch version. Some old PyTorch version causes segfault if it is imported together with onnx.

**Comment by Wheest** [comment]
Thanks for looking into this.

On a related project, I encountered an issue, which seems to be to do with how PyTorch is exporting to Onnx.

See here: https://github.com/pytorch/pytorch/issues/30560#issuecomment-573763428

My previous environment had `onnx==1.6.0`, but I downgraded to `1.5.0`.  This fixed the linked issue, as well as the issue discussed in this one.

I consider this closed, as it is PyTorch's to fix, on Onnx which have made some compatibility breaking changes?
